app/build.gradle

```diff
@@ -37,6 +37,7 @@ android {
     }
     composeOptions {
         kotlinCompilerExtensionVersion compose_version
+        kotlinCompilerVersion '1.5.21'
```

Comment:
https://flutter-developer.medium.com/android-kotlincompilerversion-is-deprecated-solved-7040db35d5a9